src/main/java/net/glowstone/i18n/LocalizedString.java

```diff
@@ -0,0 +1,13 @@
+package net.glowstone.i18n;
+
+import java.util.ResourceBundle;
+
+public interface LocalizedString {
+    ResourceBundle STRINGS = ResourceBundle.getBundle("strings");
+
```

Comment:
Add `String format(Object... args)` and a `void log(Object... args)` that knows the appropriate log level.